### DISCUSSION TITLE
Disable volume option

### DISCUSSION THREAD

**Discussion by lordoff** [DeaDBeeF-Player/deadbeef#2701] [Ideas]
Hi,
I already mentioned this idea in other discussion :-)
Is it possible to add option for disabling volume, even with alsa output?
Yes, I know that deadbeef is bit-perfect player, I liked it a lot, and honesty this is my favorite player. for many reasons. 
I don't have a idea how to implemented volume in deadbeef, but I'm afraid that is absolute must  for some poor audiophiles. :-)

**Comment by Oleksiy-Yakovenko** [comment]
Can't you simply set the volume to 100% and remove the volume slider from the UI?

**Reply by lordoff** [reply]
:-) I already read your suggestion in #1908. 
By my view, this is a placebo effect. May be similar to have disabled volume option, but...
OK, got it! Thanks Alexey!
